src/locale/kbd-model-map

```diff
@@ -66,3 +66,4 @@ lt.baltic		lt	pc105		-		terminate:ctrl_alt_bksp
 lt.l4                  lt	pc105		-		terminate:ctrl_alt_bksp
 lt			lt	pc105		-		terminate:ctrl_alt_bksp
 khmer			kh,us	pc105		-		terminate:ctrl_alt_bksp
+es-dvorak               es      microsoftpro    dvorak          terminate:ctrl_alt_bksp
```

Comment:
Could you align (at least) this line? Though, many lines in the above have spurious alignment...

Reply:
hmm weird, I'm pretty sure I used tabs to do the alignment, anyways it should be fixed now.